apps/condo/domains/organization/schema/Organization.js

```diff
@@ -18,11 +16,19 @@ const { COUNTRIES } = require('@condo/domains/common/constants/countries')
 const { DEFAULT_STATUS_TRANSITIONS } = require('@condo/domains/ticket/constants/statusTransitions')
 const { hasValidJsonStructure } = require('@condo/domains/common/utils/validation.utils')
 
+
+// This can be replaced with
+const conf = require('@core/config')
+const { LocalFileAdapter } = require('@keystonejs/file-adapters')
```

Comment:
We will use only s3 adapter

Reply:
It was an example of how to use fileadapter

Reply:
Could you please remove all non s3 adapters

Reply:
There is only one non-s3 adapter and it's a local file adapter. 
We are using a local file adapter to store images. 
I can replace the use of a local file adapter everywhere in code. But it's better to make a separate pull request
And we need a local file adapter to have a fallback if there is no sbercloud config in .env. 
I remove minio, amazon and digitalocean as they were added as an example and we do not need them

Reply:
Ok, please use one adapters everywhere :)
Because it’s not so simple to migrate from one adapter to another